powerauth-fido2-tests/src/main/java/com/wultra/security/powerauth/fido2/controller/HomeController.java

```diff
@@ -37,7 +40,7 @@
  * @author Jan Pesek, jan.pesek@wultra.com
  */
 @Controller
-@AllArgsConstructor
+@RequiredArgsConstructor
```

Comment:
Ok, but later on, we may extract `hideDeveloperOption` and other properties into e.g. `PowerAuthFido2ConfigProperties`.